app/account/views.py

```diff
@@ -40,10 +40,12 @@ def register():
     """Register a new user, and send them a confirmation email."""
     form = RegistrationForm()
     if form.validate_on_submit():
+        zip_code = ZIPCode.create_zip_code(form.zip_code.data)
         user = User(first_name=form.first_name.data,
                     last_name=form.last_name.data,
                     email=form.email.data,
-                    password=form.password.data)
+                    password=form.password.data,
+                    zip_code_id=zip_code.id)
```

Comment:
nvm, this is fine, let's keep this the way it is. I can explain the thoughts that happened if you'd like in person.